frontend-android/app/src/main/java/com/example/wetravel/views/DestinationsList.kt

```diff
@@ -42,17 +43,17 @@ fun DestinationsList(
     val destinations = listOf(
         // Add your destinations here, for example:
         Destination(
-            "MoMA", "11 W 53rd St, New York", "4.6", 50,
-            R.drawable.sample_destination_image, voted = true, numOfVotes = 5
+            UUID.randomUUID(),"MoMA", "11 W 53rd St, New York", "4.6", 50,
```

Comment:
Would this be the PlaceID taken from GoogleMapsAPI?

Reply:
@derrick-yyang good catch, I think this concern goes beyond just PR. 

I think you're right, the ID is best left to the maps API fetch. However, the catch is that we first have to be 100% sure that there wouldn't be a case where a user can add something that isn't an established place. For example, what if they just enter an address?

The destination add screen that @shanccui worked on seems to take addresses if I'm not mistaken. There's thus two ways we can go about doing this:
1. Enforce places only
2. Allow any random address — Keep UUID and generate it in our own frontend, and create another placeID field that is an optional string that is populated iff it's a place.

In case 2, the detail view can check if the destination has a place ID before going down the maps pipeline and fetching photos, names, ratings etc to display. We would need to also make ratings an optional field, and update the other list item components to deal with this.

I honestly am not sure which one is easier, @shanccui feel free to give your thoughts too about feasibility from the maps API side of things

Reply:
Ahhh I see that's a really good question actually, @shanccui is there any way to check if it's an actual "place" over some random address or pin?